documentation/modules/non-admin-permissions-for-ui.adoc

```diff
@@ -64,6 +64,10 @@ As a more comprehensive example, you can grant non-administrators the following
 |`get`, `list`, `watch`
 |`forklift.konveyer.io`
 |`forkliftcontrollers`
+
+|`create`, `patch`, `delete`
+|`forklift.konveyer.io`
```

Comment:
the API group of secrets is an empty string (it's part of the core entities)

and btw, there's a typo that repeats in this section
```suggestion
|`forklift.konveyor.io`
```

Reply:
@ahadas --should line 70 be <empty string> instead of `secrets` ?

Reply:
> @ahadas --should line 69 be "empty string" instead of `forlist.konveyor,io` ?

yep